cm/README.md

```diff
@@ -0,0 +1,54 @@
+# CM.com Integration for SMS and WhatsApp Business
+
+## Feedback and Questions
+
+As this integration is created and maintained by CM.com, the most direct way to find support is via [their fork of the project](https://github.com/CMTelecom/dialogflow-integrations).
+In that same project, you can [open issues](https://github.com/CMTelecom/dialogflow-integrations/issues) that will be read by the CM.com team of developers.
```

Comment:
Can we also mention that the fix will also be merged into this master branch? So customer will not be confused about which repo is source of the truth

Reply:
Clarified in the readme.